### PR TITLE
Custom Design for UI#243

### PR DESCRIPTION
**Pull request by hand-dot**
https://github.com/pdfme/pdfme/issues/243

### CODE REVIEW

File: playground/src/Designer.tsx [hunk 1]
```diff
@@ -36,7 +36,14 @@ function App() {
         designer.current = new Designer({
           domContainer: designerRef.current,
           template,
-          options: { font },
+          options: {
+            font,
+            customTheme: {
```
Comment (hand-dot) [inline playground/src/Designer.tsx:41]: We can use like this.

File: packages/ui/src/theme.ts
Comment (hand-dot) [inline packages/ui/src/theme.ts:1]: user can overwrite based on this defaultTheme.

File: playground/src/Designer.tsx [hunk 2]
```diff
@@ -36,7 +36,14 @@ function App() {
         designer.current = new Designer({
           domContainer: designerRef.current,
           template,
-          options: { font },
+          options: {
+            font,
+            theme: {
```
Comment (hand-dot) [inline playground/src/Designer.tsx:41]: We can use like this.

Customize Design Token: https://ant.design/docs/react/customize-theme#customize-design-token

**Comment by hand-dot** [comment]
Hey @peteward ,

If you have time, could you please check this PR?

File: packages/ui/src/components/Designer/Sidebar/index.tsx [hunk 3]
```diff
@@ -60,47 +51,32 @@ const Sidebar = (props: SidebarProps) => {
...
-                borderRadius: 2,
-                cursor: 'pointer',
-                margin: '0 auto',
-                display: 'block',
-              }}
-              onClick={addSchema}
-            >
-              <strong style={{ color: '#fff' }}>{i18n('addNewField')}</strong>
-            </button>
+            <Button type="primary" style={{ fontWeight: 600 }} onClick={addSchema}>
```
Comment (peteward) [inline packages/ui/src/components/Designer/Sidebar/index.tsx:77]: do you need to override font weights here? shouldn't the primary button style take care of this?

**Review by peteward — APPROVED** [review]
This is excellent, very exciting!

**Pull request merged**